### DISCUSSION TITLE
getPromotedProductIOS event doesn't trigger at all when opening an app from a terminated state

### DISCUSSION THREAD

**Discussion by amiceli-torstar** [hyochan/react-native-iap#1422] [General]
Posted this issue two weeks ago - hoping to get some help with this. Thanks!

https://github.com/dooboolab/react-native-iap/issues/1398

**Comment by amiceli-torstar** [comment]
It's been about 3+ weeks since I filed this - has anyone attempted/encountered this issue? Am I using the hook incorrectly? Is it incorrect for me to assume I should see promoted products when app is launched from a terminated state?

**Reply by hyochan** [reply]
Actually, this feature was developed by the community which I have not tried yet.

@ghsdh3409 @neilco May you support the current thread?

**Reply by ghsdh3409** [reply]
Oh, I also found that it is not triggered from the cold start. I will check soon!
